### ISSUE TITLE
geopackage.addGeoJSONFeatureToGeoPackage() is using hardcoded SRS

### ISSUE THREAD

**Issue by adjioev** [ngageoint/geopackage-js#93]
Hello, 

I'm trying to create a feature and store into 'rivers' geopackage file which is in 3857 projection.  It looks like `addGeoJSONFeatureToGeoPackage()` method has hardcoded SRS ->   `geometryData.setSrsId(4326);`

Is it possible to patch it with something like this:
```
featureDao.getSrs(srs, function() {
    geometryData.setSrsId(srs.srs_id);
    ...
})

```
Thanks Alex

**Comment by gotnull** [comment]
SRSID `4326` is also hardcoded in the following function:

`GeoPackage.createFeatureTableWithDataColumnsAndBoundingBox(geopackage, tableName, geometryColumns, columns, null, boundingBox, 4326, callback);`

**Comment by danielbarela** [comment]
published 1.0.23 which fixes this issue.
